dmipy/utils/utils.py

```diff
@@ -204,12 +206,16 @@ def T1_tortuosity(lambda_par, vf_intra, vf_extra=None):
     .. [3] Szafer et al. "Theoretical model for water diffusion in tissues."
         Magnetic resonance in medicine 33.5 (1995): 697-712.
     """
-    if vf_extra is None:
-        lambda_perp = (1 - vf_intra) * lambda_par
-    else:
-        fraction_intra = vf_intra / (vf_intra + vf_extra)
-        lambda_perp = (1 - fraction_intra) * lambda_par
-    return lambda_perp
+    fraction_intra = vf_intra
+    fraction_extra = 1.0 - fraction_intra if vf_extra is None else vf_extra
+
+    if S0_responses is not None:
```

Comment:
add detailed comment either here or in the doctring of the tortuosity function here - add a preliminary citation to your abstract if you want.